lib/api/v1/incident-controller.js

```diff
@@ -109,6 +139,8 @@ module.exports = function(app, user) {
         // Delete each incident explicitly to catch it in model
         incident.remove(function(err) {
           if (err) return res.send(err.status, err.message);
+          var logDetails = writelog.writeIncident(req, incident, 'deleteIncident');
```

Comment:
Could write report be a function insider the logger  library? We just pass the parameters to logger.logActivity